provider/github/review.go

```diff
@@ -40,7 +40,11 @@ func createReview(
 	requests := splitReviewRequest(req, batchReviewComments)
 	for i, req := range requests {
 		_, resp, err := client.PullRequests.CreateReview(ctx, owner, repo, number, req)
-		if err = handleAPIError(resp, err); err != nil {
+
+		if err = handleAPIError(resp, err, "review could not be pushed"); err != nil {
+			ctxlog.WithLogFields(ctx, log.Fields{
```

Comment:
is this line doing anything? you don't use the returned ctx or logger.

Reply:
This line is adding the comments to the logger `Fields`, so they will appear later when performing the real `Errorf` at [server/server.go::L114](https://github.com/src-d/lookout/pull/488/files#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bR114).

An alternative could be to get the logger `Fields` and append this new one (but when doing so, tests didn't pass because of a `nil` dereference, and I didn't want o try harder)

Reply:
I've tested your branch and yes, your code is working but it shouldn't :sweat_smile: 
The original `ctx` log fields should not be modified... https://github.com/src-d/lookout/issues/491

Reply:
fair enough. Since I didn't want to block this enhancement till we have an agreement on how to add content to the error, we can follow that in a separate PR.